microsoft/testsuites/power/common.py

```diff
@@ -72,9 +72,20 @@ def verify_hibernation(
 
     # only set up hibernation setup tool for the first time
     hibernation_setup_tool.start()
-    uptime = node.tools[Uptime]
+    # This is a temporary workaround for a bug observed in Redhat Distros
+    # where the VM is not able to hibernate immediately after installing
+    # the hibernation-setup tool.
+    # A sleep(100) also works, but we are unsure of the exact time required.
+    # So it is safer to reboot the VM.
+    if isinstance(node.os, Redhat):
```

Comment:
if only happen in redhat, you can use type(node.os) == Redhat

Reply:
done

Reply:
done